src/nyc_trees/libs/sql.py

```diff
@@ -46,13 +46,15 @@ def get_group_tree_count(group):
     return _get_count(sql, [group.pk])
 
 
-def get_total_tree_count():
+def get_total_tree_count(past_week=False):
     sql = _survey_sql + """
         SELECT COUNT(*)
         FROM survey_tree AS tree
         JOIN most_recent_survey
           ON tree.survey_id = most_recent_survey.id"""
-
+    if past_week:
+        sql += " WHERE most_recent_survey.created_at" \
+               " > CURRENT_TIMESTAMP - INTERVAL '7 days'"
```

Comment:
To be timezone agnostic you should replace `CURRENT_TIMESTAMP` with `now() at time zone 'utc'`.

If I'm reading the Postgre docs correctly, `CURRENT_TIMESTAMP` will depend upon the timezone of the DB server, which may not be UTC.